### ISSUE TITLE
[Electricraft] Batteries keep charging after their power source explode

### ISSUE THREAD

**Issue by pjolin** [ReikaKalseki/Reika_Mods_Issues#1949]
v21b

The induction generators feeding those batteries blew up and took some of the superconducting wires with them, but they're still getting power. 

That's true even after I remove the batteries and install them again next to where the cables were. 

![battery](https://user-images.githubusercontent.com/8095523/45396392-51e93900-b609-11e8-97e3-f4274d81d7b9.png)

**Comment by pjolin** [comment]
Update: it's apparently a client-server desync issue: They exploded on my client, but not on the server? 

When I started tapping the fires to extinguish them the wires, generators and shaft power bus just reappeared one by one as I touched their blocks. 

Logged off/on the server and everything was still there and powered, including collateral damage to the building being reverted. 

There was no reason (I think) for the generators to explode to start with: 8 of them were splitting half the output of a high-pressure turbine (so ~536 MW each).
